src/betterproto/__init__.py

```diff
@@ -261,7 +255,7 @@ class Enum(enum.IntEnum):
     def from_string(cls, name: str) -> int:
         """Return the value which corresponds to the string name."""
         try:
-            return cls.__members__[name]
+            return cls._member_map_[name]
```

Comment:
what's this change about?

Reply:
Accessing `enum. _member_map_ ` is a more direct call https://github.com/python/cpython/blob/master/Lib/enum.py#L365 since read-only isn't important